apps/charterafrica/src/payload/fields/post.js

```diff
@@ -0,0 +1,65 @@
+import linkGroup from "./linkGroup";
+import richText from "./richText";
+import slug from "./slug";
+
+const setPublishedOn = ({ value }) => {
+  return new Date(value);
+};
+
+const postFields = [
+  {
+    name: "title",
+    label: {
+      en: "Title",
+      fr: "Titre",
+      pt: "Título",
+    },
+    type: "textarea",
```

Comment:
```suggestion
    type: "text",
```

Title doesn't need to be that long